test/gtest/uct/test_mem.cc

```diff
@@ -209,9 +209,10 @@ UCS_TEST_P(test_mem, md_fixed) {
                     status = uct_mem_free(&uct_mem);
                 } else {
                     EXPECT_EQ(status, UCS_ERR_NO_MEMORY);
+                    munmap(curr_addr, page_size);
```

Comment:
I think mmap_fixed_address should release the memory, or use another approach - since the test maps and unmaps memory, it may still hit the issue that something else gets mapped to this range.